packages/salesforcedx-utils-vscode/src/i18n/index.ts

```diff
@@ -10,6 +10,7 @@ export {
   BASE_FILE_NAME,
   Config,
   DEFAULT_LOCALE,
+  LOCALE_JA,
```

Comment:
I'm going to create a ticket for refactoring to a single i18n service in utils so we can avoid having duplicate code in every module.